app/Services/Stack/Installers/WordPressStackInstaller.php

```
@@ -217,6 +217,112 @@ public function getInstallationTypes(): array
+        if (PHP_OS_FAMILY !== 'Windows') {
+            $uid = getmyuid();
+            $gid = getmygid();
+            if ($uid !== false && $gid !== false) {
+                $dockerCommand .= " --user {$uid}:{$gid}";
+            }
+        }
+
+        // Add image and WP-CLI command
+        $dockerCommand .= sprintf(' %s wp %s 2>&1', $image, $command);
+
+        exec($dockerCommand, $output, $exitCode);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$output'. (undefined)

(UnusedLocalVariable)